CMakeLists.txt

```diff
@@ -485,11 +496,13 @@ set_target_properties(iio PROPERTIES
 	VERSION ${VERSION}
 	SOVERSION ${LIBIIO_VERSION_MAJOR}
 	FRAMEWORK ${OSX_FRAMEWORK}
-	PUBLIC_HEADER ${LIBIIO_HEADERS}
 	C_STANDARD 99
 	C_STANDARD_REQUIRED ON
 	C_EXTENSIONS OFF
 )
+
+set_property(TARGET iio PROPERTY PUBLIC_HEADER ${LIBIIO_HEADERS})
```

Comment:
Why that change? It does the same as the set_target_properties() above.

Reply:
Because `LIBIIO_HEADERS` became a list when i append `iopp.h`. And CMake complained about multiple values when it was used as `PUBLIC_HEADER` in `set_target_properties`. So this was my workaround. Maybe there is a better way, I am no CMake expert.

Reply:
If you escape the variable with commas it works: `PUBLIC_HEADER "${LIBIIO_HEADERS}"`

Reply:
Done.